spec/models/profile_spec.rb

```diff
@@ -4,11 +4,76 @@
   let(:user) { create(:user, :signed_up) }
   let(:another_user) { create(:user, :signed_up) }
   let(:profile) { create(:profile, user: user) }
-
-  subject { profile }
+  let(:pii) do
```

Comment:
`let` and `before` are used extensively on this project.

Typically, every `context` in a rspec has a `before` block that creates the described context.